### PR TITLE
Added documentation for peer ops

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Added the documentation for the newly added functions in peer ops.

Fixes: #644

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

File: docs/BP/Ops/peer_ops.md [hunk 1]
```diff
@@ -150,4 +256,79 @@ Args:
...
+
+## 12. peer_probe_servers()
+This function probes specified servers and validate whether the probed servers are in cluster and connected state if validate is set to True.
+
+```m
+
+Args:
+    servers (str|list): A server|List of servers to be peer probed.
+    node (str): Node on which command has to be executed.
+Kwargs:
```
Comment (srijan-sivakumar) [inline docs/BP/Ops/peer_ops.md:280]: Same...not kwargs..we can call them optional parameters.

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]
I think there is a inherent confusion between kwargs and optional parameters. Simple way of explaining it is that in case of kwargs, I can pass any number of parameters to a function call with a param name while in case of optional parameters, the parameter is already defined in the function definition. It's just upto me to either use my own value or let the function use the default value.

File: docs/BP/Ops/peer_ops.md [hunk 2]
```diff
@@ -150,4 +256,79 @@ Args:
...
+## 13. peer_detach_servers()
+This function detaches a set of servers and validates the same if `validate` is set to `True`.
+
+```m
+
+Args:
+    servers (str|list): A server|List of servers to be detached.
+    node (str): Node on which command has to be executed.
+
+Kwargs:
```
[the remark this answers is not in the document]
Comment (aujjwal-redhat) [reply docs/BP/Ops/peer_ops.md:303]: yeah true. We need to modify the docstring as well.

**Review by srijan-sivakumar — APPROVED** [review]
LGTM

**Pull request merged**